decidim-debates/spec/system/user_creates_debate_spec.rb

```diff
@@ -29,6 +29,66 @@
                  settings: { taxonomy_filters: [taxonomy_filter.id] })
         end
 
+        context "and attachments are not allowed" do
+          before do
+            component_settings = component["settings"]["global"].merge!(attachments_allowed: false)
+            component.update!(settings: component_settings)
+            visit_component
+            click_on "New debate"
+          end
+
+          it "does not show the attachments form" do
+            expect(page).to have_no_css("#debate_documents_button")
+          end
+        end
+
+        context "and attachments are allowed" do
```

Comment:
Could you add a spec with what happens what if you upload a document that is not allowed? for example a ODT file ?